packages/graph/src/components/StatusIcon/StatusIcon.jsx

```diff
@@ -27,7 +27,7 @@ import {
 
 // TODO: need 'skipped' status (e.g. when expressions)
 const statusIcons = {
-  dummy: () => <></>, // eslint-disable-line react/jsx-no-useless-fragment
```

Comment:
Another one we could keep, points out opportunities for cleanup / simplification when actually wrapping content